pattern-2/bosh-release/jobs/health_check/templates/run.sh.erb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -e
+
+isAlive=0
+
+health_check() {
+    # Check if health check endpoint is alive
+    if curl --output /dev/null --silent --fail -k "$1"
+    then
+        status_code=$(curl --write-out %{http_code} --silent --output /dev/null -k ${1})
+
+        # Check if requests to the health check endpoint produces  200 response
+        if [[ "$status_code" -ne 200 ]] ; then
+            >&2 echo "WSO2 EI $2 produces an invalid response: $status_code"
+            exit 1
+        else
+            echo "WSO2 EI $2 is Running!"
+            isAlive=1
+        fi
+    else
+        >&2 echo "WSO2 EI $2 is not alive. Retrying in 10s..."
+        isAlive=0
+    fi
+}
+
+healthCheckEP=<%= p("health_check.endpoint") %>
+COUNTER=0
+
+# While the endpoint is not alive, and the server has been retrying for less than 3 minutes
+while [ ${isAlive} -eq 0 ]&&[ ${COUNTER} -lt 18 ]; do
+    sleep 10s
+    health_check ${healthCheckEP} "<%= p("health_check.name") %>"
```

Comment:
Shall we add the same change for pattern-1 as well?

Reply:
Added to Pattern 1 https://github.com/wso2/pivotal-cf-ei/pull/7/commits/adf8bf3100d136f254abbf040476814fce49f8a3